src/Adapters/Cache/InMemoryCacheAdapter.js

```diff
@@ -0,0 +1,36 @@
+import {InMemoryCache} from './InMemoryCache';
+
+export class InMemoryCacheAdapter {
+
+  constructor(ctx) {
+    this.cache = new InMemoryCache(ctx)
+  }
+
+  get(key) {
+    return new Promise((resolve, reject) => {
+      let record = this.cache.get(key);
+      if (record == null) {
```

Comment:
null and undefined are treated the same in most other systems so its best to leave it as is.

setting a key to ``null`` or ``undefined`` in my mind should delete it